plugins/symbol-processor/src/test/kotlin/org/jetbrains/dataframe/ksp/DataFrameSymbolProcessorTest.kt

```diff
@@ -210,6 +210,42 @@ class DataFrameSymbolProcessorTest {
         result.successfulCompilation shouldBe true
     }
 
+    @Test
+    fun `multi-round data schema generation`() {
+        useHostedFile(jetbrainsCsv) {
+            val result = KspCompilationTestRunner.compile(
+                TestCompilationParameters(
+                    sources = listOf(
+                        SourceFile.kotlin(
+                            "MySources.kt",
+                            """
+                @file:ImportDataSchema(name = "Repo", "$it")
+
+                import org.jetbrains.kotlinx.dataframe.DataFrame
+                import org.jetbrains.kotlinx.dataframe.annotations.DataSchema
+                import org.jetbrains.kotlinx.dataframe.annotations.ImportDataSchema
+                import org.jetbrains.kotlinx.dataframe.api.print
+                import org.jetbrains.kotlinx.dataframe.io.readJson
+
+                @DataSchema
+                interface Repos {
+                    val repositories: DataFrame<Repo>
+                }
+
+                fun main() {
+                    val df: DataFrame<Repos> = DataFrame.readJson("data/jetbrains_repositories.json") as DataFrame<Repos>
+                    df.repositories[0].print()
```

Comment:
Just a note: this code is only compiled, not executed

Reply:
Yes, I got it. I made it this way to illustrate the idea of why this multi-round generation can be needed)